python/pythonizations/SmartPtr.C

```diff
@@ -1,18 +1,18 @@
 #include "SmartPtr.h"
 
-auto_ptr<MyShareable> gime_mine() { return mine; }
-auto_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
-auto_ptr<MyShareable>& gime_mine_ref() { return mine; }
+shared_ptr<MyShareable> gime_mine() { return mine; }
+shared_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
```

Comment:
It was my first choice but it cannot be returned by `gime_mine` as easily. Using a `shared_ptr` removes the warnings maintaining the semantics of the tests intact (as far as I can tell)